next.config.js

```diff
@@ -6,42 +6,66 @@ const nextConfig = {
         protocol: 'https',
         hostname: 'assets.coingecko.com',
       },
+      {
+        protocol: 'https',
+        hostname: 'raw.githubusercontent.com',
+      },
     ],
   },
   async redirects() {
     return [
       {
-        source: '/BED',
+        source: '/(B|b)(E|e)(D|d)',
```

Comment:
https://github.com/vercel/next.js/issues/21498#issuecomment-848315717

Reply:
crazy format but love it 🤣 👌